src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2503,6 +2503,26 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return flags;
     }
 
+    til::point TermControl::_toControlOrigin(const til::point& terminalPos)

```

Comment:
You can pass `point`s by value - they're small! At just 8 bytes, they'll be passed as a register argument.